### PR TITLE
feat support of approval node of workflow

### PR DESCRIPTION
**Pull request by przemkalit**
# What does this PR do?
This PR add support of approval node which is missing while exporting the workflow.

# How should this be tested?
Manually

# Is there a relevant Issue open for this?

N/A
# Other Relevant info, PRs, etc
N/A

### CODE REVIEW

File: roles/filetree_create/templates/current_workflow_job_templates.j2 [hunk 1]
```diff
@@ -17,9 +17,17 @@ controller_workflows:
                      return_all=true, max_objects=query_controller_api_max_objects) %}
       - identifier: "{{ node.identifier }}"
         workflow_job_template: "{{ node.summary_fields.workflow_job_template.name }}"
+{% if 'approval' not in node.summary_fields.unified_job_template.unified_job_type %}
```
Comment (ivarmu) [inline roles/filetree_create/templates/current_workflow_job_templates.j2]: I would prefer `node.summary_fields.unified_job_template.unified_job_type is not match('approval')`

File: roles/filetree_create/templates/current_workflow_job_templates.j2 [hunk 2]
```diff
@@ -17,9 +17,17 @@ controller_workflows:
...
         unified_job_template: "{{ node.summary_fields.unified_job_template.name | default('ToDo: The node is pointing to a deleted Job Template') }}"
+{% endif %}
 {% if node.job_type %}
         job_type: "{{ node.job_type }}"
+{% if 'approval' in node.summary_fields.unified_job_template.unified_job_type %}
```
Comment (ivarmu) [inline roles/filetree_create/templates/current_workflow_job_templates.j2]: I would prefer `node.summary_fields.unified_job_template.unified_job_type is match('approval')`

**Review by ivarmu — APPROVED** [review]
LGTM. Just proposing two changes, but linked to my personal preferences, as your option is working as well.

**Review by adonisgarciac — APPROVED** [review]
LGTM

**Pull request merged**